### ISSUE TITLE
Change the Bookmarks dialog title to "Bookmarks and Tracks"

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#4082]
Check if there are any string contexts where "Bookmarks" without "Tracks" is still needed. If it is not required, then simply renaming the string key from "bookmarks" to "bookmarks_and_tracks" would solve the issue. If "Bookmarks" is needed somewhere in the bookmarks-only context, then the current translation should be left as-is, and a new key should be added.

1. Because we already support imported tracks from KML
2. GPX import will be added soon too.
3. Track recording is also planned.

For iOS, iphone/plist.txt file should also be updated for a long-tap menu title.

**Comment by sagar-bansal** [comment]
Hi, can I work on this issue? Thanks :)

**Comment by biodranik** [comment]
It is easy to search by key and by value and see where each string is used. No guessing and no assumptions, please.

**Comment by dmdbilal** [comment]
Can i work on this issue ?

**Comment by matkoniecz** [comment]
@dmdbilal Yes! See https://github.com/organicmaps/organicmaps/issues/4196#issuecomment-1373856560

> We do not assign issues to first-time contributors. Just choose any issue and fix it.

**Comment by subhajit20** [comment]
@biodranik  should I change that also in this file iphone/plist.txt file ?

**Comment by biodranik** [comment]
Only if you test it and see that it is a relevant change.

**Comment by Naveed1891** [comment]
i want to work on it

**Comment by Jean-BaptisteC** [comment]
Fixed by #5242

**Comment by biodranik** [comment]
@Jean-BaptisteC There are other places in the app where now "bookmarks" word is used. It would be great to add "tracks" where necessary. FAQ is also an example that should be corrected.

**Comment by oleg-rswll** [comment]
Related to #5680
